cid/config.py

```diff
@@ -9,7 +9,7 @@
 DATABASE_URLS = {
     # NOTE(mhayden): The four slashes in the production database path ensures
     # that an absolute path is used.
-    "production": "sqlite:////code/cid.db",
```

Comment:
This will probably break our current production container setup.
With the ubi image we can't write to / by default.
Maybe move it somewhere else?

Reply:
Oops, that was a quick test I did to fix something and shouldn't have been in this PR. I'll fix that.